src/MySqlConnector/Core/Row.cs

```diff
@@ -567,4 +518,5 @@ private static void CheckBufferArguments<T>(long dataOffset, T[] buffer, int buf
 	private readonly int[] m_dataOffsets;
 	private readonly int[] m_dataLengths;
 	private ReadOnlyMemory<byte> m_data;
+	private IColumnReader[] columnReaders;
```

Comment:
`private readonly IColumnReader[] m_columnReaders;`

Will fix in https://github.com/mysql-net/MySqlConnector/pull/1331/commits/1cc81efbb218269a98eaf23e4e53ae469a707e40.